openapi/sms/v1/instructions.md

```diff
@@ -0,0 +1,244 @@
+# Instructions
+
+SAP Emarsys empowers marketers around the world to create truly personalized omnichannel experiences that deliver business outcomes. The Emarsys SMS Partner API allows partners to integrate into the Emarsys SMS channel. The partner API is REST based.
+
+## API Specification
+
+The API specification has been created using OpenAPI 3.0.3. It is available to be exported using the links below:
+
+- [Partner Service Specification](../partner-service.yaml)
```

Comment:
```suggestion
- [Partner Service Specification](./partner-service.yaml)
```

Reply:
we navigate to the parent directory here because in stoplight markdown files are places in a `docs` directory (within the root) whereas the OpenAPI yaml files are directly in the root